### DISCUSSION TITLE
Audio Recording setting for wav File

### DISCUSSION THREAD

**Discussion by SpitFire-Code** [katspaugh/wavesurfer.js#3051] [General]
Hello, I trying to create a player which records a wav file in 11025khz, 8 Bit Mono. What settings should i use in wavesurfer to get the desired file?

**Comment by katspaugh** [comment]
It should be possible to set a sample rate on a media stream but it’s currently not passed in the plugin options.
For reference: https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints
